website/docs/UserEvent.md

```diff
@@ -28,3 +28,39 @@ Creates User Event instances which can be used to trigger events.
 ### Options
 - `delay` - controls the default delay between subsequent events, e.g. keystrokes, etc.
 - `advanceTimers` - time advancement utility function that should be used for fake timers. The default setup handles both real and Jest fake timers.
+
+
+## `press()`
+
+```ts
+type(
+  element: ReactTestInstance,
+): Promise<void>
+```
+
+Example
+```ts
+const user = userEvent.setup();
+
+await user.press(touchable);
+```
+
+This helper simulates a press on any pressable element, e.g. Pressable, Text or TextInput. Unlike fireEvent.press which is a simpler API that will only call the onPress prop, this simumlates the entire press event in a more realistic way by reproducing what really happens when a user presses a pressable component. This means for instance for onPressIn and onPressOut props will also be invoked.  
```

Comment:
```suggestion
This helper simulates a press on any pressable element, e.g. `Pressable`, `TouchableOpacity`, `Text`, `TextInput`, etc. Unlike `fireEvent.press()` which is a simpler API that will only call the `onPress` prop, this simulates the entire press event in a more realistic way by reproducing what really happens when a user presses an interface view. This will trigger additional events like `pressIn` and `pressOut`. 
```